_data/accessibility-tests/file-input.yml

```diff
@@ -0,0 +1,103 @@
+title: File input
+component_status: pass
+test_items:
+# General tests
+  - summary: File input works on mobile devices.
+    summary_additional: When you use a mobile device or tablets, file inputs should work in both portrait and landscape views with no loss of functionality.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.3.4
+  - summary: Color is not the only method used to convey meaning.
+    summary_additional: When you view the file input component, focus indicators, text labels, field descriptions and messaging all help establish meaning.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.1
+  - summary: File input text meets color contrast requirements.
+    summary_additional: When you use a color contrast checker to view the file input, contrast between the text color and background color is at least 4.5:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.3
+  - summary: File input elements have proper contrast.
+    summary_additional: When you use a color contrast analyzer, the contrast between the file input and all of its elements is at least 3:1 from their respective surrounding backgrounds.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.11
+  - summary: Error text indicates an incorrect file type.
+    summary_additional: When you drag an incorrect file type into the file upload box, text appears describing the error.
+    test_status: pass
+    test_type: screen_reader
```

Comment:
**issue:** Need to update test type
```suggestion
    test_type: general
```

Reply:
Resolved [a293494](https://github.com/uswds/uswds-site/pull/2963/commits/a293494611d9afbfb645af8e11a55592761789cf)